lib/index.js

```diff
@@ -478,7 +478,7 @@ module.exports = {
 				},
 				schema: [
 					{
-						enum: [ "allow", "always", "nested", "never", ],
+						enum: [ "never-global", "always", "nested", "never", ],
```

Comment:
just for consistency, let's move "never-global" to the end of the list, since it's being added, and since it's more related to "never" than to "always". :)